test/support/generator/random-value-generator.ts

```diff
@@ -20,10 +25,10 @@ export class RandomValueGenerator {
     return this.chance.url({ protocol: 'https' });
   }
 
-  stringOfNumericCharacters(minLength?: number): string {
+  stringOfNumericCharacters(minLength?: number, maxLength?: number): string {
     const stringOptions: Partial<Chance.StringOptions> = { pool: '0123456789' };
-    if (minLength) {
-      const length = this.chance.integer({ min: minLength, max: Math.max(20, minLength * 2) });
+    if (minLength || maxLength) {
```

Comment:
I think it's fine to delete the `if` statement here, because if the caller doesn't specify max or min then it's fine for our default to be "generate a string between 0 and 20 in length"